crates/biome_css_analyze/src/lint/nursery/no_invalid_direction_in_linear_gradient.rs

```diff
@@ -50,39 +50,20 @@ declare_rule! {
 
 lazy_static! {
     // It is necessary to find case-insensitive string.
-    // For example, both 'linear-gradinet' and 'Linear-gradient' should pass the check.
-    pub static ref LINEAR_GRADIENT_FUNCTION_NAME: Regex =
-        Regex::new(r"^(?i)(-webkit-|-moz-|-o-|-ms-)?linear-gradient").unwrap();
-
-    // It is necessary to find case-insensitive string.
-    // Also Check if 'in' is a word.
+    // Also Check if 'in' is a word boundary.
     // For examples,`to top in srgb` is valid but `to top insrgb` is not valid.
     pub static ref IN_KEYWORD: Regex = Regex::new(r"(?i)\bin\b").unwrap();
```

Comment:
This regular expression is used to check if "in" is a word.
**examples**
**valid**
- `90deg in oklab`
- `in oklab`
- `90 deg in`

**invalid**

- `90degin oklab`
- `inoklab`